spec/lib/job_notifier/notifier_spec.rb

```diff
@@ -127,7 +127,7 @@ def perform_with_feedback
 
   context "without defining perform_with_feedback" do
     before do
-      class ImageUploadJob < ActiveJob::Base
+      class ImageUploadJob < ApplicationJob
```

Comment:
⚠️ **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Lint/ConstantDefinitionInBlock: Do not define constants this way within a block.